src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Accordion/BitAccordion.scss

```
@@ -6,48 +6,167 @@
-        background-color: $clr-bg-dis;
+    // The colors the header is painted with while the pointer is over it and while it is held down. They are
+    // declared as tokens rather than written into the header rule, because the shade that reads as "a little
+    // darker" depends on the background the accordion carries - see the Background classes below.
+    --bit-acd-hov: #{$clr-bg-pri-hover};
+    --bit-acd-prs: #{$clr-bg-pri-active};
+
+    // The duration of the expand/collapse transition. TransitionDuration arrives inline as the -full token
+    // and everything below reads this derived one, following the -full pattern of motion.fluent.scss, so the
+    // reduced-motion query can still collapse a duration that was set inline. Without one it falls back to
+    // the theme's own long duration.
+    //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)